deepchem/models/torch_models/grover.py

```diff
@@ -92,3 +98,312 @@ def forward(self, graph_batch: List[torch.Tensor]):
                 "atom_from_bond": output[1][0],
                 "bond_from_bond": output[1][1]
             }
+
+
+class _BondVocabPredictionLayer(nn.Module):
```

Comment:
Let's make this layer public. It might be useful for other models. Can you also add it to the docs?

Reply:
Fixed in #3277